### PR TITLE
SCRUM-601 Removing leading & trailing extra spaces from input string

### PR DESCRIPTION
**Pull request by kthorat-prog**

### CODE REVIEW

File: src/main/cliapp/src/components/DiseaseAnnotationsComponent.js [hunk 1]
```diff
@@ -163,6 +165,7 @@ export const DiseaseAnnotationsComponent = () => {
             itemTemplate={subjectItemTemplate}
             completeMethod={searchSubject}
             onChange={(e) => onSubjectEditorValueChange(props, e)}
+            forceSelection={true}
```
Comment (adamgibs) [inline src/main/cliapp/src/components/DiseaseAnnotationsComponent.js:168]: Setting forceSelection as a property leads to a bug where if you tab away from an input, it causes the cell to go blank.

**Review by adamgibs — COMMENTED** [review]
This didn't seem to work for me. When I paste in an item with whitespace and try to submit, it still get an error. Maybe it also needs to be trimmed somewhere in the onRowEditSave function?

File: src/main/cliapp/src/components/DiseaseAnnotationsComponent.js (hunk 1 above)
Comment (kthorat-prog) [reply src/main/cliapp/src/components/DiseaseAnnotationsComponent.js:168]: Ohh yes, I accidently added that while fixing other issue. Only the .replace lines (2 times) is the change.

**Comment by kthorat-prog** [comment]
> This didn't seem to work for me. When I paste in an item with whitespace and try to submit, it still get an error. Maybe it also needs to be trimmed somewhere in the onRowEditSave function?

**Comment by adamgibs** [comment]
I just get the "not a valid entry" message when I try to submit after pasting.

**Comment by kthorat-prog** [comment]
Ohh...I forgot to take care of that one... Thanks. Will fix it and then create a PR.

**Pull request closed without merging**